core-bundle/tests/Contao/Database/ResultTest.php

```diff
@@ -25,16 +26,34 @@ public function testEmptyResult(): void
 
         /** @var Result|object $result */
```

Comment:
This type-hint is causing the PHPStan error I believe.

Reply:
fixed in 0fd67ddfb98ce85785aa11e79e3163c8e0fb9d68